### PR TITLE
Implement KEEP_EXISTING_ACKS_AND_READS Action for Card Updates (#6181)

### LINKED ISSUE #6181: Implement KEEP_EXISTING_ACKS_AND_READS Action for Card Updates
Description:

In certain scenarios, we need to update cards without requiring the end user to re-read or acknowledge the card. To accommodate this, we propose the addition of a new action in cards: KEEP_EXISTING_ACKS_AND_READS.

When this action is set and the card already exists (i.e., it's an update), the update should preserve the existing read and acknowledgment statuses from the current card.

Testing:

- Create a Karate test to verify that the action is correctly persisted in the data.
- Unit tests should cover the logic of preserving the acknowledgment and read statuses.
- No Cypress tests are required for this issue.

Documentation:

Once implemented, this new action should be documented in https://opfab.github.io/documentation/current/reference_doc/#_actions_actions.
 Additionally, the Swagger definition file should be updated to include this new action.

### PR DESCRIPTION
**Pull request by quinarygio**
Fix #6181 

- In release note :
  -  In chapter :  Features
  -  Text : #6181 : Implement KEEP_EXISTING_ACKS_AND_READS Action for Card Updates

### CODE REVIEW

File: services/cards-publication/src/main/java/org/opfab/cards/publication/services/CardNotificationService.java [hunk 1]
```diff
@@ -45,19 +45,21 @@ private void pushCardInEventBus(CardOperation cardOperation) {
             eventBus.sendEvent("card", mapper.writeValueAsString(cardOperation));
 
             LightCard card = cardOperation.card();
-            log.debug(
-                    "Card operation sent to eventbus, type={}, ids={}, cards={}, groupRecipients={}, entityRecipients={}, userRecipients={}",
+            log.info(
```
Comment (freddidierRTE) [inline services/cards-publication/src/main/java/org/opfab/cards/publication/services/CardNotificationService.java]: Keep log in mode debug

File: services/cards-publication/src/main/java/org/opfab/cards/publication/model/LightCard.java [hunk 2]
```diff
@@ -35,6 +36,8 @@ public record LightCard(
         String titleTranslated,
         String summaryTranslated,
         List<TimeSpan> timeSpans,
+        @JsonIgnore
```
Comment (freddidierRTE) [inline services/cards-publication/src/main/java/org/opfab/cards/publication/model/LightCard.java]: I do not understand why it shall be ignore

File: services/cards-consultation/src/main/java/org/opfab/cards/consultation/services/CardSubscriptionService.java [hunk 3]
```diff
@@ -262,19 +263,34 @@ private void processNewCard(String cardOperationAsString, CardSubscription subsc
...
+            publishCardDelete(cardOperation, subscription);
+        }
+    }
+
+    private void publishCard(JSONObject cardOperation, CardSubscription subscription) {
+        if (cardOperation.get("type").equals("UPDATE")) { //for the front an update is considered as an ADD
+            cardOperation.put("type", "ADD");
+
+            JSONObject cardObj = (JSONObject) cardOperation.get("card");
+            JSONArray usersAcks =(JSONArray)cardOperation.get("usersAcks");
```
Comment (freddidierRTE) [inline services/cards-consultation/src/main/java/org/opfab/cards/consultation/services/CardSubscriptionService.java]: usersAcks should be get from card ?

**Review by freddidierRTE — CHANGES_REQUESTED** [review]

File: services/cards-publication/src/main/java/org/opfab/cards/publication/model/LightCard.java (hunk 2 above)
Comment (quinarygio) [reply services/cards-publication/src/main/java/org/opfab/cards/publication/model/LightCard.java]: Because it is not needed on frontend side

**Review by freddidierRTE — APPROVED** [review]

**Pull request merged**